drivers/dac/dac_demo/dac_demo.c

```diff
@@ -85,6 +92,128 @@ int32_t dac_demo_remove(struct dac_demo_desc *desc)
 	return SUCCESS;
 }
 
+
+int32_t dac_get_number_of_channels(void* dev)
```

Comment:
Maybe:

`int32_t dac_get_number_of_channels(void* dev, uint32_t *ch_nr)`
`{`
` struct dac_demo_desc *desc = dev;`
` *ch_nr = dev->active_ch;`
` return SUCCESS;`
`}`

In this manner you will keep the same format as other functions, returning an error code.

Reply:
removed the function completely as it was no longer used

Reply:
same mentions for `int32_t adc_get_number_of_channels(void* dev)`
https://github.com/analogdevicesinc/no-OS/pull/935/files#diff-de54365b8e51917ba102b742e5f00709bd9026a55d14db2e564c12dd67384d31R115